src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -48,6 +48,16 @@ public function testFiles()
         $this->assertIterator($this->toAbsolute(array('foo/bar.tmp', 'test.php', 'test.py', 'foo bar')), $finder->in(self::$tmpDir)->getIterator());
     }
 
+    public function testRemoveTrailingSlash()
+    {
+        $finder = $this->buildFinder();
+
+        $expected = $this->toAbsolute(array('foo/bar.tmp', 'test.php', 'test.py', 'foo bar'));
+        $in = '//'.realpath(self::$tmpDir).'//';
```

Comment:
adding `//` at the beginning of a Windows absolute path won't produce a valid path. So this test must be skipped on Windows.

Reply:
Thanks. Fixed